zeebe/gateway-grpc/src/main/java/io/camunda/zeebe/gateway/interceptors/impl/AuthenticationHandler.java

```diff
@@ -63,7 +71,12 @@ public Either<Status, Context> authenticate(final String authorizationHeader) {
                 .withCause(e));
       }
 
-      return Either.right(Context.current().withValue(USER_CLAIMS, token.getClaims()));
+      final Map<String, Object> claims = new HashMap<>(token.getClaims());
+      claims.put(
+          Authorization.AUTHORIZED_USERNAME,
+          claims.get(oidcAuthenticationConfiguration.getUsernameClaim()));
+
+      return Either.right(Context.current().withValue(USER_CLAIMS, claims));
```

Comment:
:x: This isn't quite correct. The `USER_CLAIMS` context is used later to pass along actual user claims, see `ClaimTransformer#applyUserClaim`. That means our `authorized_username` claim becomes `user_token_authorized_username`, indistinguishable from a normal user claim that happens to be named `authorized_username`.
I think what we should do instead is copy from the basic auth handler and set the `USERNAME` context.